.env.development

```diff
@@ -3,5 +3,6 @@ NEXT_PUBLIC_BASE_URL=https://powerful-brushlands-00932.herokuapp.com
 NEXT_PUBLIC_BOT_USERNAME=upperdev_bot
 NEXT_PUBLIC_REGISTER_BOT_USERNAME=upper_test_bot
 NEXT_PUBLIC_ARTICLE_BUCKET_URL=https://upper-dev-article-img-bucket.s3.ap-south-1.amazonaws.com/
+NEXT_PUBLIC_TUTORIAL_BUCKET_URL=https://upper-dev-tutorial-img-bucket.s3.ap-south-1.amazonaws.com/
```

Comment:
@musojon1 this PR has 98 file changes. It seams you have created the branch from dev. You should create it from main. Close this PR, create a new branch from main, cherry pick your changes (https://git-scm.com/docs/git-cherry-pick) and open a new PR